### ISSUE TITLE
Poor accuracy while running the pretrained_word_embeddings example

### ISSUE THREAD

**Issue by Acheron2012** [keras-team/keras#9104]
I followed the tutorial from "https://blog.keras.io/using-pre-trained-word-embeddings-in-a-keras-model.html" and I downloaded the the code of pretrained_word_embeddings.py(included glove.6B.100d.txt and news20.tar.gz).
When the program was finished, I found that the accuracy was only 73% while the tutorial said that it could reach 95% classification accuracy on the validation set after only 2 epochs. 
I don't know what was wrong, how can I solve this?
Thank you.

**Comment by barnybug** [comment]
Seeing the same issue - the best it reaches is val_acc: 0.74 or so, and then ends up overfitting (train acc increasing, val_acc going down) if run for >10 epochs.

Adding in a `Dropout(0.5)` layer before the final preds only slows the overfitting a bit, and doesn't improve that validation accuracy.

Would be great @fchollet input on this one?

**Comment by ThomasWMarshall** [comment]
I'm also seeing this issue. Would love to know if anyone has figured out why.

**Comment by ThomasWMarshall** [comment]
This issue persists.  @fchollet Do you have any ideas?
